.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
we generally don't commit ide/editor configs, do we make an exception for this one? I'd be fine with it in this specific case.

Reply:
thaaat was actually not on purpose 😅 I guess we remove it, that config won't help anyone whoe doesn't have a paid grammarly subscription + vscode extension soooo

Reply:
I removed it